EduNLP/SIF/segment/segment.py

```diff
@@ -84,6 +88,7 @@ def __init__(self, item, figures: dict = None):
         self._ques_mark_segments = []
         self._tag_segments = []
         self._sep_segments = []
+        self._textf_segments = []
```

Comment:
Do not add textf segment. Remove the style info in textf string, e.g. 'b' in \textf{'xxx', 'b'} and add 'xxx' into text.

For instance, "hello world, I am \textf{Robot, b}" should be aggregate into one text_segments as ['hello world, I am Robot']

Reply:
OK I'll commit again.
Plus, as the style info should be removed at the very beginning, why provide `\textf` in SIF?